### PR TITLE
Project Completed

### PR DESCRIPTION
**Pull request by bouaik**
## Restaurant page

- Single Page Application
- Tab Switching functionality
- Filtering items
- Contact Us page

Stylelint is passing locally but here I'm not sure what's the issue, if you see the Stylelint report you can see it says Run npx stylelint "** when it should run stylelint "**/*.{css,scss}", if you have any idea how I can solve this problem please let me know.

### CODE REVIEW

File: .github/workflows/linters.yml [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+      - uses: actions/checkout@v2
+      - uses: actions/setup-node@v1
+        with:
+          node-version: "12.x"
+      - name: Setup Stylelint
+        run: |
+          npm install --save-dev stylelint@13.3.x stylelint-scss@3.17.x stylelint-config-standard@20.0.x stylelint-csstree-validator
+          [ -f .stylelintrc.json ] || wget https://raw.githubusercontent.com/microverseinc/linters-config/master/javascript/.stylelintrc.json
+      - name: Stylelint Report
+        run: npx stylelint "**
```
Comment (wathigo) [inline .github/workflows/linters.yml]: This line does not look exactly as specified in the Microverse's repo.
Let's change it to:
```yml
run: npx stylelint "**.{css,scss}"
```

**Review by wathigo — CHANGES_REQUESTED** [review]
### Changes Required
Great job on this project :clap: 
You have currently implemented all the modules and handle DOM manipulation.
Let's fix the linter config files in order to fix the failing linter check.

**REQUIRED**
* Linters config: Do not make any changes in config files - they represent style guidelines that you share with your team - which is a group of all Microverse students. ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/general/linter_config.png) from Pathwright). See the comments for more info :point_down: 

**After making the changes, submit another request for review.**

**Happy Hacking :computer: 
Simon Wathigo.**

File: .github/workflows/linters.yml (hunk 1 above)
Comment (bouaik) [reply .github/workflows/linters.yml]: Thank you sir

**Review by mikenath223 — APPROVED** [review]
### CONGRATULATIONS  👏👏👏

This project meets the requirements for approval.

**STATUS: APPROVED ✅**


Don't forget to fill the project completion form👍
Good luck with the next project.

HAPPY CODING👨‍💻

**Pull request merged**